desktop/ui/src/main/java/org/datacleaner/util/DatastoreCreationUtil.java

```diff
@@ -57,7 +57,7 @@ protected static FileDatastoreEnum getDatastoreTypeFromFile(File file) {
             final String extension = FilenameUtils.getExtension(file.getName());
 
             for (FileDatastoreEnum datastoreType : EnumSet.allOf(FileDatastoreEnum.class)) {
-                if (datastoreType._extensions.contains(extension)) {
+                if (datastoreType._extensions.contains(extension.toLowerCase())) {
```

Comment:
Are there any risks that "extension" variable is null? I could imagine that FilenameUtils would return null if the filename does not have any dot in it.